### DISCUSSION TITLE
Lock error

### DISCUSSION THREAD

**Discussion by ljensen51** [homebridge-simplisafe3/homebridge-simplisafe3#237] [General]
Getting this error on my lock:

[9/7/2021, 3:53:14 PM] [homebridge-simplisafe3] This plugin generated a warning from the characteristic 'Lock Current State': characteristic value expected valid finite number and received "undefined" (undefined). See https://git.io/JtMGR for more info.

**Comment by shamoon** [comment]
Hi @ljensen51 , thanks for reporting this. Odd, I have two locks and dont see this behavior. As noted in the link from the warning:

1. Is it still working for you? (If so, safe to ignore the warning)
2. If you want to help out a little more, can you [enable debug](https://github.com/homebridge-simplisafe3/homebridge-simplisafe3#debug) (Homebridge + plugin) and send the full output? Maybe will help track this down.

**Comment by ljensen51** [comment]
Yes, the lock is working fine.  I did do a debug, but am not sure if there’s anything in there that I should delete before sending, like the ’sid’ or ’serial’ values.  Let me know what to delete and I’ll send it.

The error seems to appear when the lock is unlocked and I restart Homebridge.  If the lock is locked when I restart Homebridge, I do not see that error.

Thank you.

Message from Lyle Jensen
***@***.***

**Reply by shamoon** [reply]
Aha! That helps, cheers. Should be able to sort this out.

**Comment by shamoon** [comment]
Cheers @ljensen51 this should be sorted in 1.7.1, thanks for reporting

**Comment by ljensen51** [comment]
Good morning Michael,

Yes, v1.7.1 of the plugin has resolved the undefined error on my SS lock!

This morning with v1.7.0 still installed I first unlocked the lock and restarted HB.  This reproduced the undefined error.

Then I updated to v1.7.1 and restarted HB.  There were no errors from the lock.

Thank you!

Message from Lyle Jensen
***@***.***
